packages/report-server/src/reportBuilder/context/extractContextProps.ts

```diff
@@ -32,7 +32,7 @@ export const extractContextProps = (transform: unknown): ContextProp[] => {
     return [];
   }
 
-  const expressions = (transform as TransformParams[])
+  const expressions = transform
```

Comment:
This type cast wasn't mandatory, so decided to remove it and get rid of the `TransformParams` dependency